src/components/JourneyTimeline/JourneyTimeline.tsx

```diff
@@ -24,6 +24,51 @@ const expandControlSpacingPx = 100;
 
 const stringifyTime = (time) => globalStringifyTime(time, { use12Hour: true });
 
+const isInfillStation = (stationId, route) => {
+    const alwaysInfills = new Set([
+        "place-rr-everett-jct",
+        "place-rr-revere-center",
+        "place-rr-south-salem",
+        "place-rr-tufts-university",
+        "place-rr-montvale-avenue",
+        "place-rr-umass-lowell",
+        "place-rr-rourke-bridge",
+        "place-rr-willow-springs",
+        "place-rr-nashua",
+        "place-rr-merrimack",
+        "place-rr-manchester-center",
+        "place-rr-brickbottom",
+        "place-rr-clematis-brook",
+        "place-rr-weston-128",
+        "place-rr-ceylon-park",
+        "place-rr-river-street",
+        "place-rr-west-station",
+        "place-rr-newton-corner",
+        "place-rr-pawtucket",
+        "place-rr-hingham-depot",
+        "place-rr-cohasset-center",
+        "place-rr-braintree-highlands",
+        "place-rr-bridgewater-center",
+        "place-rr-mboro-centre-st",
+        "place-rr-weymouth-col-sq",
+        "place-rockland-n-abington",
+        "place-kingston-jct",
+        "place-plymouth-center",
+    ]);
+    const infillInRoute = {
```

Comment:
@idreyn do we have a list of these somewhere? I know the ones for the northern lines, but I'm not as familiar with the southern lines

Reply:
I'm not sure if it's comprehensive but I tried to put them in italics in the spec doc!

Reply:
Aside from the ones that are existing rapid transit stations, I believe they're all in `infill_stations.py` in the schedule generator. Also, I meant to make them all start with `place-rr-` and I must have slipped up on the last few. We could easily fix that and then use the convention of `place-rr-` (or `place-infill-`) implying an infill station.